### DISCUSSION TITLE
GPS Sync for Hero 8

### DISCUSSION THREAD

**Discussion by thomaspingel** [gopro/labs#263] [Feature Requests]
Hi - I love the GPS sync feature on the Hero 9.  Is there a way to enable this on the 8?  Is there a technical reason why it doesn't work for this camera?

**Comment by dnewman-gpsw** [comment]
Camera architectures evolve, this feature was simply too tricky to port back to HERO8.  It was prototyped on 9 and 10, and improved on 11.

**Comment by thomaspingel** [comment]
I understand.  How has it improved for 11?  I'm considering getting a newer one so I could use this feature, but had planned on just getting  a cheaper 9.  What makes the 11 better with respect to GPS sync?

**Reply by dnewman-gpsw** [reply]
HERO11 advantages are explained here https://gopro.github.io/labs/control/gpssync/
